{"url": "https://www.reddit.com/r/learnprogramming/comments/9a8p7j/passing_a_dictionary_as_an_instance_of_a_class/", "text": "New to Python and trying to get this task done but I keep getting this error . \n\nThere's a class called \"Menu\" with various parameters and I created an instance of this class and passed in the values but I keep getting this error. The dictionary  called \"items\" is probably the one causing the issue \n\n    class Menu(object):\n      def __init__(self,name,items,start_time,end_time):\n        self.name = name\n        self.items = items\n        self.start_time = start_time # 24 hr format\n        self.end_time = end_time     # 24 hr format\n        \n    items = { 'pancakes':7.50,'waffles':9.00,'burger':11.00,'home fries':4.50,'coffee':11.50,'espresso':3.00,'tea':1.00,'mimosa':10.50,'orange juice':3.50}\n    \n    brunch = Menu('Brunch',items,11,16)   \n    print(brunch)\n    \n\nThis is the error I am getting:\n\n    <__main__.Menu object at 0x7fec6b46c668>\n\n&#x200B;", "score": 1, "comments": [{"body": "Thats not an error. brunch is pointing to an object called \"Menu\". so when you print out brunch it says its an object. You can access its content like this\n\n    print(brunch.name)\n    print(brunch.items)", "id": "e4tk33i", "replies": [{"body": "Okay I see now and I was thinking that it's not printing the result as this was an error. Anyways thanks man", "id": "e4tlpke", "replies": []}]}], "title": "Passing a dictionary as an instance of a class"}